src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function createDiv(divClass) {
+  const div = document.createElement('div');
+
+  div.className = divClass;
+
+  if (divClass === 'message') {
+    div.textContent = 'Promise was resolved!';
+  } else {
+    div.textContent = 'Promise was rejected!';
+  }
+
+  return div;
+}
+
+const promise1 = new Promise(resolve => {
+  logo.addEventListener('click', () => resolve(createDiv('message')));
+});
+
+const promise2 = new Promise(
+  (resolve, reject) => setTimeout(() => {
+    reject(createDiv('error-message'));
+  }, 3000));
+
+promise1.then(result => document.body.append(result));
```

Comment:
According to task description you should:
_add success and error handlers for both promises_